client/tsconfig.json

```diff
@@ -39,6 +39,7 @@
     "noPropertyAccessFromIndexSignature": true,
     "esModuleInterop": true,
     "skipLibCheck": true,
-    "forceConsistentCasingInFileNames": true
+    "forceConsistentCasingInFileNames": true,
```

Comment:
what was this one for?

Reply:
just linting adding a trailing comma.
the option is a handy catch for any changes made on windows I guess [reference](https://www.typescriptlang.org/tsconfig#forceConsistentCasingInFileNames)